CI run failed

```
Error:  2022-07-20 15:47:35.866 [main] o.a.s.t.i.f.watcher.ITWatcher - Error case: assertExecute[proxy: dbtbl_with_readwrite_splitting_and_encrypt -> MySQL -> Literal -> SHOW DATABASES], message: java.lang.AssertionError: 
Expected: is "information_schema"
     but: was "mysql"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertObjectValue(BaseDALIT.java:104)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertRow(BaseDALIT.java:88)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertRows(BaseDALIT.java:75)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertResultSet(BaseDALIT.java:47)
	at org.apache.shardingsphere.test.integration.engine.dal.GeneralDALIT.assertExecute(GeneralDALIT.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Error:  2022-07-20 15:47:35.866 [main] o.a.s.t.i.f.watcher.ITWatcher - Error case: assertExecute[proxy: dbtbl_with_readwrite_splitting_and_encrypt -> MySQL -> Literal -> SHOW DATABASES], message: java.lang.AssertionError: 
Expected: is "information_schema"
     but: was "mysql"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertObjectValue(BaseDALIT.java:104)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertRow(BaseDALIT.java:88)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertRows(BaseDALIT.java:75)
	at org.apache.shardingsphere.test.integration.engine.dal.BaseDALIT.assertResultSet(BaseDALIT.java:47)
	at org.apache.shardingsphere.test.integration.engine.dal.GeneralDALIT.assertExecute(GeneralDALIT.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:55)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.044 s <<< FAILURE! - in org.apache.shardingsphere.test.integration.engine.dal.GeneralDALIT
Error:  assertExecute[proxy: dbtbl_with_readwrite_splitting_and_encrypt -> MySQL -> Literal -> SHOW DATABASES](org.apache.shardingsphere.test.integration.engine.dal.GeneralDALIT)  Time elapsed: 0.028 s  <<< FAILURE!
java.lang.AssertionError: 
Expected: is "information_schema"
     but: was "mysql"
	at org.apache.shardingsphere.test.integration.engine.dal.GeneralDALIT.assertExecute(GeneralDALIT.java:53)
[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    GeneralDALIT.assertExecute:53->BaseDALIT.assertResultSet:47->BaseDALIT.assertRows:75->BaseDALIT.assertRow:88->BaseDALIT.assertObjectValue:104 
Expected: is "information_schema"
     but: was "mysql"
[INFO] 
Error:  Tests run: 196, Failures: 1, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- maven-source-plugin:3.2.1:jar-no-fork (attach-sources) @ shardingsphere-integration-test-suite ---
[INFO] Building jar: /home/runner/work/shardingsphere/shardingsphere/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/target/shardingsphere-integration-test-suite-5.1.3-SNAPSHOT-sources.jar
[INFO] 
[INFO] --- apache-rat-plugin:0.14:check (default) @ shardingsphere-integration-test-suite ---
[INFO] RAT will not execute since it is configured to be skipped via system property 'rat.skip'.
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.0:verify (integration-tests) @ shardingsphere-integration-test-suite ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:20 min
[INFO] Finished at: 2022-07-20T15:47:37Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.apache.maven.plugins:maven-failsafe-plugin:2.22.0:verify (integration-tests) on project shardingsphere-integration-test-suite: There are test failures.
Error: 
Error:  Please refer to /home/runner/work/shardingsphere/shardingsphere/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/target/failsafe-reports for the individual test results.
Error:  Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
Error:  -> [Help 1]
Error: 
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error: 
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error: Process completed with exit code 1.
```

--- a/examples/shardingsphere-proxy-example/shardingsphere-proxy-distsql-example/src/main/resources/server/server.yaml
+++ b/examples/shardingsphere-proxy-example/shardingsphere-proxy-distsql-example/src/main/resources/server/server.yaml
@@ -55,7 +55,6 @@ props:
   proxy-hint-enabled: true
   sql-show: false
   check-table-metadata-enabled: false
-  lock-wait-timeout-milliseconds: 50000 # The maximum time to wait for a lock
   show-process-list-enabled: false
     # Proxy backend query fetch size. A larger value may increase the memory usage of ShardingSphere Proxy.
   # The default value is -1, which means set the minimum value for different JDBC drivers.

--- a/shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/props/ConfigurationPropertyKey.java
+++ b/shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/config/props/ConfigurationPropertyKey.java
@@ -78,11 +78,6 @@ public enum ConfigurationPropertyKey implements TypedPropertyKey {
     SHOW_PROCESS_LIST_ENABLED("show-process-list-enabled", String.valueOf(Boolean.FALSE), boolean.class, false),
     
     /**
-     * The length of time in milliseconds an SQL waits for a global lock before giving up.
-     */
-    LOCK_WAIT_TIMEOUT_MILLISECONDS("lock-wait-timeout-milliseconds", String.valueOf(50000L), long.class, false),
-    
-    /**
      * Proxy backend query fetch size. A larger value may increase the memory usage of ShardingSphere Proxy.
      * The default value is -1, which means set the minimum value for different JDBC drivers.
      */

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/dbtbl_with_readwrite_splitting/mysql/show_databases.xml
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/dbtbl_with_readwrite_splitting/mysql/show_databases.xml
@@ -20,8 +20,8 @@
         <column name="schema_name" />
     </metadata>
     <row values="dbtbl_with_readwrite_splitting" />
+    <row values="mysql" />
     <row values="information_schema" />
     <row values="performance_schema" />
-    <row values="mysql" />
     <row values="sys" />
 </dataset>

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/dbtbl_with_readwrite_splitting_and_encrypt/mysql/show_databases.xml
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/dbtbl_with_readwrite_splitting_and_encrypt/mysql/show_databases.xml
@@ -20,8 +20,8 @@
         <column name="schema_name" />
     </metadata>
     <row values="dbtbl_with_readwrite_splitting_and_encrypt" />
+    <row values="mysql" />
     <row values="information_schema" />
     <row values="performance_schema" />
-    <row values="mysql" />
     <row values="sys" />
 </dataset>

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/readwrite_splitting/mysql/show_databases.xml
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dal/dataset/readwrite_splitting/mysql/show_databases.xml
@@ -19,9 +19,9 @@
     <metadata>
         <column name="schema_name" />
     </metadata>
-    <row values="information_schema" />
-    <row values="performance_schema" />
     <row values="readwrite_splitting" />
     <row values="mysql" />
+    <row values="information_schema" />
+    <row values="performance_schema" />
     <row values="sys" />
 </dataset>

